<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="Main">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>241</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="mainClient">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="text">
            <string>Client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mainServer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="text">
            <string>Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mainForwarder">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="text">
            <string>Forwarder</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Client">
       <widget class="QPushButton" name="startClient">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="clientForwardIP">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>192.168.0.8</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="clientForwardPort">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>7001</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="clientDestPort">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>8000</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="webViewer">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>611</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="fileWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>361</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="secondGo">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>430</y>
          <width>99</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TCP Retrieve</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>UDP Retrieve</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>13</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Forwarder IP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>53</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>TCP Port</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>93</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>UDP Port</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Server">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>TCP Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>7000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>UDP Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="udpPortEdit">
             <property name="text">
              <string>8888</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="serverStart">
           <property name="text">
            <string>Start Server</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Forwarder">
       <widget class="QPushButton" name="startForward">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>510</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Forward</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="networkView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>501</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
